api/v1beta1/azuremanagedcluster_conversion.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The Kubernetes Authors.
+Copyright 2023 The Kubernetes Authors.
```

Comment:
Generally I've been told not to change the initial copyright assertion of a file because lawyers. If there's a need to reassert copyright, you usually add it: `Copyright 2021, 2023 The Kubernetes Authors.`

Having said that, IANAL and I'm not asking for this to be reverted because 🤷🏻.